aten/src/ATen/native/transformers/attention.cpp

```diff
@@ -760,8 +760,8 @@ _scaled_dot_product_flash_attention_cpu(
   int64_t num_head = query.size(1);
   int64_t headSize = query.size(3);
 
-  TORCH_CHECK(c10::isFloatingType(dtype) && dtype != ScalarType::Half,
-    "scaled_dot_product_attention_flash_attention: Expected data type in FP32, FP64, BF16, but got ", dtype, " instead.");
+  TORCH_CHECK(c10::isFloatingType(dtype),
```

Comment:
Q: is FP8 considered floating type?

Reply:
ahh this is a good point it seems that c10::isFloatingtype also returns true for fp8 types we should ban those, I didn't realize the set isFloatingType included  those, good catch!